log/term/colorlogger.go

```diff
@@ -0,0 +1,149 @@
+package term
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"sync"
+
+	"github.com/go-kit/kit/log"
+)
+
+// Color is the abstract color, the zero value is the Default.
+type Color uint8
+
+const (
+	NoColor = Color(iota)
+	Red
+	Green
+	Yellow
+	Blue
+	Magenta
+	Cyan
+	White
+	Default
+
+	maxColor
+)
+
+var resetColorBytes = []byte("\x1b[0m")
+var fgColorBytes [][]byte
+var bgColorBytes [][]byte
+
+func init() {
+	for color := NoColor; color < maxColor; color++ {
+		fgColorBytes = append(fgColorBytes, []byte(fmt.Sprintf("\x1b[%dm", 30+color)))
+		bgColorBytes = append(bgColorBytes, []byte(fmt.Sprintf("\x1b[%dm", 40+color)))
+	}
+}
```

Comment:
@tgulacsi That's not going to work for me as a maintainer of this package because my development environment doesn't have `sh`, `echo`, `sed`, or `xargs`.